### ISSUE TITLE
Layout for non-threaded

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/mail#3657]
For mails which are not a thread but just single emails, the layout is a bit strange currently cause it shows too much info. Also it makes no sense to be able to collapse the only email.

So we could check what to do with the header there (possibly just not show the people bubbles in the header?) and probably block collapsing the mail.

**Comment by ChristophWurst** [comment]
Hi @jancborchardt,
Is this still relevant? If so, can you provide mockup for the expected layout?

**Comment by jancborchardt** [comment]
No, this seems to be outdated by now, nice! :) It’s not even possible anymore to collapse the single message.
